components/nav.css

```diff
@@ -0,0 +1,52 @@
+.top-nav {
+  display: flex;
+  flex-wrap: wrap;
+  min-width: 100%;
+  background-color: var(--white);
+  color: var(--sec-text);
+  padding: 2rem;
+  align-items: center;
+  font-size: 1.6rem;
+}
+
+.top-nav .navigation {
+  flex-grow: 1;
+  align-items: center;
+  display: flex;
+}
+.navigation .nav-brand {
+  font-weight: 500;
+  font-size: 2rem;
+  color: var(--primary-color);
+  padding: 0 3rem;
+}
+.nav-col {
+  display: flex;
+}
+.nv-col-1 {
+  width: 100%;
+}
+.nv-col-2 {
+  position: absolute;
+  left: 40%;
+  gap: 5px;
+}
+@media (max-width: 767px) {
```

Comment:
Ideally the screen breakpoint should be 768px for max-width, 769px for min-width
refer devtools mobile responsiveness

Reply:
ok, will make change